scripts/combat.gd

```diff
@@ -453,7 +453,9 @@ class combatant:
 			var skill = globals.abilities.abilitydict[i]
 			var newbutton = scene.get_node("grouppanel/skilline/skill").duplicate()
 			scene.get_node("grouppanel/skilline").add_child(newbutton)
-			var cost = globals.spells.spellCostCalc(skill.costmana)
+			var cost = skill.costmana
```

Comment:
This file is reverted to a less concise form and auto-skill is removed.

Reply:
fixed for next commit